src/main/java/com/aws/greengrass/config/ConfigurationWriter.java

```diff
@@ -210,6 +210,7 @@ private synchronized void truncateTlog() {
         logger.atDebug(TRUNCATE_TLOG_EVENT).log("existing tlog writer closed");
```

Comment:
This log should be after the old writer is actually closed.

Reply:
moved after close()